lib/icinga/scheduleddowntime.cpp

```diff
@@ -88,7 +88,13 @@ void ScheduledDowntime::TimerProc()
 {
 	for (const ScheduledDowntime::Ptr& sd : ConfigType::GetObjectsByType<ScheduledDowntime>()) {
 		if (sd->IsActive() && !sd->IsPaused())
```

Comment:
I think you backported an old version of the PR branch, the latest one has curly braces for this if.